src/app/mytools/mytools.service.ts

```diff
@@ -128,8 +128,12 @@ export class MytoolsService extends MyEntriesService<DockstoreTool, OrgToolObjec
     });
   }
 
-  getPath(entry: DockstoreTool): string {
-    return entry.tool_path || '';
+  getPath(entry: DockstoreTool | Workflow): string {
+    if (MytoolsService.isWorkflowBasedObject(entry)) {
+      return entry.full_workflow_path;
+    } else {
+      return entry.tool_path || '';
```

Comment:
I think it's harmless the way you have it, but can tool_path be null/undefined?

Reply:
I'm not sure how it would be tbh.. But I should have put entry.full_workflow_path || '' to be consistent since that is what the other getPath function has. Changing